spec/system/diaper_drive_participant_system_spec.rb

```diff
@@ -69,15 +69,15 @@
       create(:donation, :with_items, created_at: 1.week.ago, item_quantity: 15, source: Donation::SOURCES[:diaper_drive], diaper_drive_participant: @ddp)
     end
 
-    scenario "Existing Diaper Drive Participants show in the #index with some summary stats" do
+    it "shows existing Diaper Drive Participants in the #index with some summary stats" do
       visit url_prefix + "/diaper_drive_participants"
       expect(page).to have_xpath("//table/tbody/tr/td", text: @ddp.business_name)
       expect(page).to have_xpath("//table/tbody/tr/td", text: "25")
     end
 
-    scenario "Single Diaper Drive Participants show semi-detailed stats about donations from that diaper drive" do
+    it "allows single Diaper Drive Participants to show semi-detailed stats about donations from that diaper drive" do
       visit url_prefix + "/diaper_drive_participants/#{@ddp.to_param}"
-      expect(page).to have_xpath("//table/tr", count: 3)
+      expect(page).to have_xpath("//tr", count: 3)
```

Comment:
oh! I think the reason this might have been failing was because I bet it wants to be `"//table/tbody/tr"` instead

If it's passing now, that's fine though